app/views/categories/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="mx-auto md:w-2/3 w-full flex">
+  <div class="mx-auto">
+    <% if notice.present? %>
```

Comment:
this block should be moved to application.html.erb but i believe that we already have it

Reply:
fixed